Formula/goreman.rb

```diff
@@ -0,0 +1,42 @@
+require "language/go"
+
+class Goreman < Formula
+  desc "Foreman clone written in Go"
+  homepage "https://github.com/mattn/goreman"
+  url "https://github.com/mattn/goreman/archive/v0.1.1.tar.gz"
+  sha256 "d234ae81a397ab04338f26e30db79811715202413e0a3b9ff20395df7df675df"
+
+  depends_on "go"
```

Comment:
This is likely a `build` time only dep.

Reply:
Can you explain what or how I need to modify the code to address this issue?